presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -103,7 +103,7 @@ Example literals: ``DECIMAL '10.3'``, ``DECIMAL '1234567890'``, ``1.1``
 .. note::
 
     For compatibility reasons decimal literals without explicit type specifier (e.g. ``1.2``)
-    are treated as values of the ``DOUBLE`` type by default up to version 0.198. 
+    are treated as values of the ``DOUBLE`` type by default up to version 0.198.
```

Comment:
Just specify the current behavior. It doesn't matter what happened many versions ago.